src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -79,12 +75,7 @@ public boolean canUnload() {
 	@Override
 	public void unload() {
 		super.unload();
-		unloaded = true;
-
-		while (!dialogues.isEmpty()) {
-			// When disposed the dialogue is removed from the list
-			dialogues.get(0).dispose();
-		}
+		httpsInfoTabsPanel.removeAll();
```

Comment:
The whole method could be removed (this statement is not required, the parent panel is automatically removed).
Although we do need to remove all tabs when the session is changed? (At least that's how normally all status panels behave.)

Reply:
So I need to implement/add sessionChangedListener? Then on change remove all?

Reply:
I think so.